ui/app/pages/routes/routes.component.js

```diff
@@ -142,7 +143,11 @@ export default class Routes extends Component {
           component={SendTransactionScreen}
           exact
         />
-        <Authenticated path={SWAPS_ROUTE} component={Swaps} />
+        {provider.chainId === MAINNET_CHAIN_ID ? (
+          <Authenticated path={SWAPS_ROUTE} component={Swaps} />
+        ) : (
+          <Authenticated path={DEFAULT_ROUTE} component={Home} />
```

Comment:
Why is an additional entry for `Home` being added in this case?

Reply:
Instead of conditionally omitting the Swaps component here in the router, I'd suggest returning a `<Redirect />` to Home from the top-level swaps component instead. That's how we handle this sort of case elsewhere.

As-written this is very broken, as the duplicate `Home` entry will prevent any routes below here from working.

Reply:
@Gudahtt this is now fixed